frontend/sw360-portlet/src/main/resources/content/Language.properties

```diff
@@ -210,14 +210,16 @@ ClearingRequestPriority-CRITICAL=Clearing time less than 1 week.
 ClearingRequestPriority-HIGH=Clearing time greater than 1 week but less than 2 weeks.
 ClearingRequestPriority-LOW=Clearing time greater than 4 weeks.
 ClearingRequestPriority-MEDIUM=Clearing time greater than 2 weeks but less than 4 weeks.
-ClearingRequestState=New: CR is pending to be Accepted by the Clearing Team.&#10A sanity check for the artifacts required for Clearing will be done prior to Accepting a CR. &#10Accepted: CR has passed SanityCheck and is accepted into the clearing queue. &#10Rejected: CR has failed SanityCheck. Contact Clearing Team. (Rejecting resets the CR priority) &#10In Queue: CR is next in queue for clearing. &#10In Progress: CR is being worked on by the clearing team. &#10Closed: Clearings requested in the CR is complete. (Closing resets the CR priority) &#10Awaiting Response: Clearing Team is waiting for input/action from the Requesting User.
+ClearingRequestState=New: CR is pending to be Accepted by the Clearing Team.&#10A sanity check for the artifacts required for Clearing will be done prior to Accepting a CR. &#10Pending Input: Inputs to perform clearing is incomplete. Clearing team cannot&#10process the CR to next stage until project provides the missing inputs.&#10Sanity Check: The Clearing team is performing the sanity check on inputs provided by project team.&#10Source code availability for all linked releases are checked during this stage. &#10Accepted: CR has passed SanityCheck and is accepted into the clearing queue. &#10Rejected: CR has failed SanityCheck. Contact Clearing Team. (Rejecting resets the CR priority) &#10In Queue: CR is next in queue for clearing. &#10In Progress: CR is being worked on by the clearing team. &#10Closed: Clearings requested in the CR is complete. (Closing resets the CR priority) &#10Awaiting Response: Clearing Team is waiting for input/action from the Requesting User.
```

Comment:
Pending Input: 
The input needed for clearing is incomplete. The clearing team cannot proceed to the next stage of processing the CR until the project provides the missing input.
Sanity Check:
The Clearing team is conducting a sanity check on the inputs provided by the project team. At this stage, they verify the availability of the source code for all linked releases

Please change the titles accordingly.